bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/TimingWindowDialog.java

```diff
@@ -89,9 +89,9 @@ public void insertUpdate(DocumentEvent e) {
 		};
 
 		window.setValue(new Date());
-		duration.setValue(MS_PER_HOUR * 123 + MS_PER_MINUTE * 45);
```

Comment:
It just displayed "123.45". I don't think there was any particular reason for this.